src/account/AccountPickScreen.js

```diff
@@ -15,6 +15,9 @@ const styles = StyleSheet.create({
   accountList: {
     flex: 0,
   },
+  addAccountButton: {
+    margin: 5,
+  },
```

Comment:
This would work, but it needs an explanatory comment.

Reply:
Thank you @ray-kraesig ,
 from https://github.com/zulip/zulip-mobile/commit/b0730a0636862f09d195c33be09d47776e449080, 

>and some sort of fix so that this bug doesn't recur.

I have  tried implementing "nestedScrollEnabled", but it has not worked properly.( https://reactnative.dev/docs/scrollview )
 in this commit,in Screen component, children which has flatList/SectionList is wrapped in View,
if not they are wrapped in ScrollView.

Reply:
Hello @ray-kraesig, can you review this commit?
if it is ok I will Squash them into 2 commits as you have suggested